.circleci/config.yml

```diff
@@ -0,0 +1,61 @@
+# Copyright 2019 Stanford University see LICENSE file
+# CircleCI 2.0 configuration file
```

Comment:
i think line 2 is sorta obvious, but 🤷

Reply:
this was actually copypasta from the sinopia_acl circle config that i ripped off, but happy to remove if it irks you.